sw/host/opentitanlib/src/test_utils/load_sram_program.rs

```diff
@@ -26,20 +29,50 @@ use top_earlgrey::top_earlgrey_memory;
 pub struct SramProgramParams {
     /// Path to the VMEM file to load.
     #[structopt(long)]
-    pub vmem: PathBuf,
+    pub vmem: Option<PathBuf>,
+
+    /// Path to the ELF file to load.
+    #[structopt(long)]
```

Comment:
should this and the VMEM option have a `conflicts_with`? I.e., do you only want a user to provide either:
1. a VMEM file plus a load_addr, OR
2. an ELF file?

Reply:
Ideally yes but when I tried, I ran into a structopt internal error :-/ Since structopt is in maintenance and the plan is to switch to clap eventually, I just added code that makes sure the user provides sensible input (hopefully got that right).

Reply:
I'll try again just to make sure.

Reply:
ah got it, no worries.

Reply:
I wasn't able to reproduce the problem, I was using a combination of `requires` and `conflicts_with`. In any case, the combination in the latest version seems to work.